src/renderer/utils/createStorage.ts

```diff
@@ -1,73 +1,211 @@
-// Helper to work with browser's local/session storage api
-
-export interface IStorageHelperOptions {
-  addKeyPrefix?: boolean;
-  useSession?: boolean; // use `sessionStorage` instead of `localStorage`
+// Helper for work with persistent local storage (default: window.localStorage)
+
+import { app, remote } from "electron";
+import path from "path";
+import { ensureDir, readJson, writeJson } from "fs-extra";
+import type { CreateObservableOptions } from "mobx/lib/api/observable";
+import { action, comparer, observable, reaction, toJS, when } from "mobx";
+import produce, { Draft, enableMapSet, setAutoFreeze } from "immer";
+import { isEqual, isFunction, isObject, noop } from "lodash";
+
+setAutoFreeze(false); // allow to merge observables
+enableMapSet(); // allow merging maps and sets
+
+export function createStorage<T>(key: string, defaultValue?: T, observableOptions?: CreateObservableOptions) {
+  return new StorageHelper<T>(key, {
+    autoInit: true,
+    storage: jsonFileStorageAdapter as StorageAdapter<T>,
+    observable: observableOptions,
+    defaultValue,
+  });
 }
 
-export function createStorage<T>(key: string, defaultValue?: T, options?: IStorageHelperOptions) {
-  return new StorageHelper(key, defaultValue, options);
+export type StorageModel = Record<string, any>; // any json-compatible model
+
+export interface StorageAdapter<T = StorageModel> {
```

Comment:
This doesn't really do what I think you mean. Plus I don't think it should matter to the consumers how to data is serialized.
```suggestion
export interface StorageAdapter<T> {
```